src/1d/flagcheck.f

```diff
@@ -20,7 +20,7 @@ subroutine flagcheck(rectflags,ilo,ihi,mbuff,iflags,
 c
 
       do 10 i = imin, imax
-        if (rectflags(i) .eq. goodpt) go to 10
+        if (rectflags(i) .le. DONTFLAG) go to 10
         if (iflags(i) .ne. 1) then  !point not nested. turn off
              rectflags(i) = 0.
```

Comment:
should we instead set
    `rectflags(i) = DONTFLAG`
for consistency and clarity?